deeplake/util/spinner.py

```diff
@@ -1,9 +1,9 @@
-from deeplake.constants import SPINNER_START_DELAY, SPINNER_ENABLED
```

Comment:
as we are not using `SPINNER_ENABLED` can you please remove it from constants

Reply:
It is used below